scripts/basic-workflow-tests.sh

```diff
@@ -39,7 +39,13 @@ set_defaults() {
     fi
 
     NUM_INSTRUCTIONS=1
-    GENERATE_ARGS+=("--num-cpus" "$(nproc)")
+	# nproc doesn't exist on MacOS, so we have to use its equivalent:
+	# `sysctl -n hw.logicalcpu`: https://github.com/memkind/memkind/issues/33#issuecomment-540614162
+	if [[ "$(uname)" == "Darwin" ]]; then
```

Comment:
the indenting is off.  You are introducing tabs into a file that uses spaces to indent.  Please keep it consistent with how it was.

Reply:
Oh great catch, my editor displays them identically. Thanks!